pkg/imagebuilder/content.go

```diff
@@ -41,8 +41,11 @@ func New() *ConfigBuilder {
 	// TODO: needs appropriate value if AUTH_TYPE != none
 	pullSecretToken := getEnv("PULL_SECRET_TOKEN", "")
 
-	serviceBaseURL := fmt.Sprintf("http://%s/",
-		net.JoinHostPort(nodeZeroIP, "8090"))
+	serviceBaseURL := url.URL{
+		Scheme: "http",
+		Host:   net.JoinHostPort(nodeZeroIP, "8090"),
+		Path:   "/",
```

Comment:
Looks like this adds extra `/` and results in `BASE_URL="http://192.168.111.80:8090//api/assisted-install/v2"` in `/usr/local/bin/start-cluster-installation.sh  which then breaks auto-start cluster installation.
https://github.com/openshift-agent-team/fleeting/blob/main/data/ignition/files/usr/local/bin/start-cluster-installation.sh#L8